packages/pcsx-rearmed/build.sh

```diff
@@ -0,0 +1,24 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/notaz/pcsx_rearmed
+TERMUX_PKG_DESCRIPTION="Yet another PCSX fork based on the PCSX-Reloaded project"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=23
+TERMUX_PKG_SRCURL=git+https://github.com/notaz/pcsx_rearmed
+TERMUX_PKG_GIT_BRANCH=r${TERMUX_PKG_VERSION}
+TERMUX_PKG_SHA256=887e9b5ee7b8115d35099c730372b4158fd3e215955a06d68e20928b339646af
+TERMUX_PKG_DEPENDS="libpng, mesa-dev, pulseaudio, sdl, zlib"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_pre_configure() {
+	CFLAGS+=" -fno-integrated-as"
+}
```

Comment:
`no-integrated-as` does not work in this way. Please call `termux_setup_no_integrated_as` function instead. Sorry for the ill-documentedness. Examples are in `build.sh` of `libgcrypt` and `libpixman`.